qml/keys/languages.js

```diff
@@ -50,6 +50,7 @@ function languageIdToName(languageId)
     if (languageId == "pt")         return i18n.tr("Portuguese");
     if (languageId == "ro")         return i18n.tr("Romanian");
     if (languageId == "ru")         return i18n.tr("Russian");
+    if (languageId == "sc")         return i18n.sc("Sardinian");
```

Comment:
This should be `i18n.tr,` not `.sc`. The Sardinian keyboard shows up as a blank spot in the list of keyboards at the moment.